packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/FlipbookViewer/components/FlipbookControls.js

```diff
@@ -67,6 +67,34 @@ const FlipbookControls = ({
 
   const [iterationCounter, setIterationCounter] = useState(0)
 
+  /** DefaultLayout for classify page styling has breakpoints at 700px and 1160px
+   * In FlipbookControls we're simply checking for when this component is < 450px
+   * which happens both before and after the 700px layout breakpoint
+   */
+  const [smallScreenStyle, setSmallScreenStyle] = useState(false)
+  const controlsContainer = useRef(null)
+  const resizeObserver = useRef(null)
+
+  useEffect(() => {
```

Comment:
You might be able to replace this hook with:
```js
const { width, height, ref } = useResizeDetector
```
from `react-resize-detector`. We use that elsewhere in the monorepo to monitor the width of an element and react as the screen gets smaller.

Reply:
It's only in app-project and it's not longer maintained for 4+ years, so I opted to go with a solution we can easily adapt to a custom hook in lib-classifier if we'd like to.

Reply:
The version that we're using was published 5 months ago: https://www.npmjs.com/package/react-resize-detector

Reply:
To be honest, it's probably the same as your code here.

Reply:
You're right! I was thinking of `react-resize-observer` somehow. I'll leave this resize detector as is for now to avoid install another package, but I'll keep in mind that the other FEM packages are using an external package.